InvenTree/templates/InvenTree/settings/plugin_settings.html

```diff
@@ -0,0 +1,112 @@
+{% extends "panel.html" %}
+{% load i18n %}
+{% load inventree_extras %}
+{% load plugin_extras %}
+
+{% block label %}plugin-{{plugin_key}}{% endblock %}
+
+
+{% block heading %}
+{% blocktrans with name=plugin.human_name %}Plugin details for {{name}}{% endblocktrans %}
+{% endblock %}
+
+{% block content %}
+
+<div class="row">
+    <div class="col-md-6">
+        <h4>{% trans "Plugin information" %}</h4>
+        <table class='table table-striped table-condensed'>
+            <col width='25'>
+            <tr>
+                <td></td>
+                <td>{% trans "Name" %}</td>
+                <td>{{ plugin.human_name }}{% include "clip.html" %}</td>
+            </tr>
+            <tr>
+                <td><span class='fas fa-user'></span></span></td>
+                <td>{% trans "Author" %}</td>
+                <td>{{ plugin.author }}{% include "clip.html" %}</td>
+            </tr>
+            <tr>
+                <td><span class='fas fa-calendar-alt'></span></td>
+                <td>{% trans "Date" %}</td>
+                <td>{{ plugin.pub_date }}{% include "clip.html" %}</td>
+            </tr>
+            {% if plugin.version %}
+            <tr>
+                <td><span class='fas fa-hashtag'></span></td>
+                <td>{% trans "Version" %}</td>
+                <td>{{ plugin.version }}{% include "clip.html" %}</td>
+            </tr>
+            {% endif %}
+            {% if plugin.website %}
+            <tr>
+                <td><span class='fas fa-globe'></span></td>
+                <td>{% trans "Website" %}</td>
+                <td>{{ plugin.website }}{% include "clip.html" %}</td>
+            </tr>
+            {% endif %}
+        </table>
+
+        {% if plugin.is_package == False %}
+        <p>{% trans 'The code information is pulled from the latest git commit for this plugin. It might not reflect official version numbers or information but the actual code running.' %}</p>
+        {% endif %}
+    </div>
+    <div class="col-md-6">
+        <h4>{% trans "Package information" %}</h4>
+        <table class='table table-striped table-condensed'>
+            <col width='25'>
+            <tr>
+                <td></td>
+                <td>{% trans "Installation method" %}</td>
+                <td>
+                    {% if plugin.is_package %}
+                    {% trans "This plugin was installed as a package" %}
+                    {% else %}
+                    {% trans "This plugin was found in a local InvenTree path" %}
+                    {% endif %}
+                </td>
+            </tr>
+            {% if plugin.is_package == False %}
```

Comment:
Does `is_package` signify that this was installed via PIP? Or is there another way of installing an external plugin?

Reply:
it was installed via pip or a equivilant tool. It is a PEP so anybody can implement the mechanic. easy_install for example